var/spack/repos/builtin/packages/netcdf-c/package.py

```diff
@@ -181,6 +182,8 @@ def configure_args(self):
             else:
                 config_args.append('--disable-parallel4')
 
+            config_args += self.enable_or_disable('jna')
```

Comment:
This option seems to be available starting version `4.3.2` and the oldest listed version is `4.3.3`. Any particular reason for putting this only `if self.spec.satisfies('@4.4:')`? Otherwise, I would add this argument unconditionally.

Reply:
It might be worth putting in the logic to make sure its only added on the proper version. Just in the off chance someone adds/needs an older version. I can go ahead and add it. 

Should I make it so it should be `if self.spec.satisfies('@4.3.2:')`? I don't see any harm in doing that, and it would prepare it for if older versions were ever added.